ocs_ci/ocs/ocp.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         resource_name="",
         selector=None,
         field_selector=None,
+        threading_lock=None,
```

Comment:
Please move it as a last one parameter, if someone is using positional arguments, this way you can easily break it.